helsingborg-stad/app-mitt-helsingborg
# Fix/restore xcode project

## Explain the changes you’ve made

Restored xCode project.

## Explain why these changes are made

To be able to release a Testflight build for user tests.

## How to test the changes?
No changes should be noted beside new home screen icon.
Start the app with command "yarn ios" and check if everything starts as it should.

## Was this feature tested in the following environments?
